packages/runtime/datastore-definitions/.eslintrc.cjs

```diff
@@ -4,8 +4,8 @@
  */
 
 module.exports = {
+	extends: ["@fluidframework/eslint-config-fluid/strict", "prettier"],
```

Comment:
In other packages, we put `extends` at the top of the config, so I moved this to match.